run/gust.node8.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -l
+#PBS -N cm1
+#PBS -l select=8:ncpus=128:mpiprocs=128:ompthreads=1
+#PBS -l walltime=04:00:00
+#PBS -j oe
+#PBS -q main@gusched01
+#PBS -A UNDM0006
+
+
+export PALS_NRANKS=1024
+export PALS_PPN=128
+export PALS_DEPTH=1
+export PALS_CPU_BIND=depth
+
+#mpiexec ./cm1.exe --namelist namelist.figureE.cpu >& node=8.figureE.512x512.log
+#mpiexec ./cpu.exe --namelist namelist.figureD.cpu >& node=8.figureD.256.log
+mpiexec ./cpu.exe --namelist namelist.figureA.cpu >& node=8.figureA.256.log
```

Comment:
Which compiler did you use for the CPU build/run? I tried the `nvhpc` compiler but the execution failed. Probably we need to load the corresponding modules here as well, similar to the GPU script.

Reply:
I am using the standard environment and the nvhpc/22.7 compiler.  What kind of execution failure are you getting?

Reply:
Here is the error message from the output log file:
**file-transfer RPC timeout after 120s, received 0 of 1 responses**.

Reply:
I am using `nvhpc/22.7` as well and I do not load any CUDA module for the CPU run.

Reply:
Aha, I see your point now. I need to do "module unload cuda" in the run script even if it is for a CPU run. Now the CPU case is running on Gust.

Reply:
I would suggest to putting the following commands to your CPU run script:
```
module purge
module load nvhpc
module load cray-mpich
module load ncarcompilers
module unload cuda
```